### ISSUE TITLE
Use DoExit instead of Shutdown

### ISSUE THREAD

**Issue by Luatan** [thijsvanloef/palworld-server-docker#395]
**Is your feature request related to a problem? Please describe.**
We are using the rcon shutdown command to stop the server. With this it always shows a message in game chat, that the server will shutdown in 1 second.

**Describe the solution you'd like**
Choose a command without output, and use the broadcast command to inform the players.

My suggestion would be to use the DoExit command instead. As far as I can tell it is the same just without the delay and message.

**Describe alternatives you've considered**
Keep the shutdown command as is.

---

What do you guys think?

**Comment by Dashboy1998** [comment]
> My suggestion would be to use the DoExit command instead. As far as I can tell it is the same just without the delay and message.

When I first introduced the safe shutdown I was unsure if DoExit was a hard or soft termination so I went with shutdown. I'm not sure how to prove if DoExit is the same as shutdown with no timer.

**Comment by Luatan** [comment]
> > My suggestion would be to use the DoExit command instead. As far as I can tell it is the same just without the delay and message.
> 
> When I first introduced the safe shutdown I was unsure if DoExit was a hard or soft termination so I went with shutdown. I'm not sure how to prove if DoExit is the same as shutdown with no timer.

I just tested both of them. They both trigger these log messages
```
palworld-server  | dlmopen steamservice.so failed: steamservice.so: cannot open shared object file: No such file or directory
palworld-server  | Shutdown handler: cleanup.
```

Because of this I would argue they do the same.

**Comment by Dashboy1998** [comment]
> > > My suggestion would be to use the DoExit command instead. As far as I can tell it is the same just without the delay and message.
> > 
> > 
> > When I first introduced the safe shutdown I was unsure if DoExit was a hard or soft termination so I went with shutdown. I'm not sure how to prove if DoExit is the same as shutdown with no timer.
> 
> I just tested both of them. They both trigger these log messages
> 
> ```
> palworld-server  | dlmopen steamservice.so failed: steamservice.so: cannot open shared object file: No such file or directory
> palworld-server  | Shutdown handler: cleanup.
> ```
> 
> Because of this I would argue they do the same.

I tested using `inotifywait -m -e modify,create,delete palworld/Pal/Saved` to see the same files are edited with both commands.

Both edit all the INI files so I think it's safe to change it.
```
Setting up watches.  Beware: since -r was given, this may take a while!
Watches established.
palworld/Pal/Saved/Config/LinuxServer/ MODIFY Compat.ini
palworld/Pal/Saved/Config/LinuxServer/ MODIFY Compat.ini
palworld/Pal/Saved/Config/LinuxServer/ MODIFY Paper2D.ini
palworld/Pal/Saved/Config/LinuxServer/ MODIFY Paper2D.ini
palworld/Pal/Saved/Config/LinuxServer/ MODIFY ControlRig.ini
palworld/Pal/Saved/Config/LinuxServer/ MODIFY ControlRig.ini
palworld/Pal/Saved/Config/LinuxServer/ MODIFY LiveLink.ini
palworld/Pal/Saved/Config/LinuxServer/ MODIFY LiveLink.ini
palworld/Pal/Saved/Config/LinuxServer/ MODIFY Composure.ini
palworld/Pal/Saved/Config/LinuxServer/ MODIFY Composure.ini
palworld/Pal/Saved/Config/LinuxServer/ MODIFY ConcertSyncCore.ini
palworld/Pal/Saved/Config/LinuxServer/ MODIFY ConcertSyncCore.ini
palworld/Pal/Saved/Config/LinuxServer/ MODIFY EditorScriptingUtilities.ini
palworld/Pal/Saved/Config/LinuxServer/ MODIFY EditorScriptingUtilities.ini
palworld/Pal/Saved/Config/LinuxServer/ MODIFY EnhancedInput.ini
palworld/Pal/Saved/Config/LinuxServer/ MODIFY EnhancedInput.ini
palworld/Pal/Saved/Config/LinuxServer/ MODIFY DatasmithContent.ini
palworld/Pal/Saved/Config/LinuxServer/ MODIFY DatasmithContent.ini
palworld/Pal/Saved/Config/LinuxServer/ MODIFY GLTFExporter.ini
palworld/Pal/Saved/Config/LinuxServer/ MODIFY GLTFExporter.ini
palworld/Pal/Saved/Config/LinuxServer/ MODIFY VariantManagerContent.ini
palworld/Pal/Saved/Config/LinuxServer/ MODIFY VariantManagerContent.ini
palworld/Pal/Saved/Config/LinuxServer/ MODIFY CommonUI.ini
palworld/Pal/Saved/Config/LinuxServer/ MODIFY CommonUI.ini
palworld/Pal/Saved/Config/LinuxServer/ MODIFY FullBodyIK.ini
palworld/Pal/Saved/Config/LinuxServer/ MODIFY FullBodyIK.ini
palworld/Pal/Saved/Config/LinuxServer/ MODIFY Niagara.ini
palworld/Pal/Saved/Config/LinuxServer/ MODIFY Niagara.ini
palworld/Pal/Saved/Config/LinuxServer/ MODIFY MediaIOFramework.ini
palworld/Pal/Saved/Config/LinuxServer/ MODIFY MediaIOFramework.ini
palworld/Pal/Saved/Config/LinuxServer/ MODIFY ApexDestruction.ini
palworld/Pal/Saved/Config/LinuxServer/ MODIFY ApexDestruction.ini
palworld/Pal/Saved/Config/LinuxServer/ MODIFY Metasound.ini
palworld/Pal/Saved/Config/LinuxServer/ MODIFY Metasound.ini
palworld/Pal/Saved/Config/LinuxServer/ MODIFY Synthesis.ini
palworld/Pal/Saved/Config/LinuxServer/ MODIFY Synthesis.ini
palworld/Pal/Saved/Config/LinuxServer/ MODIFY CameraCalibrationCore.ini
palworld/Pal/Saved/Config/LinuxServer/ MODIFY CameraCalibrationCore.ini
palworld/Pal/Saved/Config/LinuxServer/ MODIFY LiveLinkCamera.ini
palworld/Pal/Saved/Config/LinuxServer/ MODIFY LiveLinkCamera.ini
palworld/Pal/Saved/Config/LinuxServer/ MODIFY ImGui.ini
palworld/Pal/Saved/Config/LinuxServer/ MODIFY ImGui.ini
palworld/Pal/Saved/Config/LinuxServer/ MODIFY Wwise.ini
palworld/Pal/Saved/Config/LinuxServer/ MODIFY Wwise.ini
palworld/Pal/Saved/Config/LinuxServer/ MODIFY PalWorldSettings.ini
palworld/Pal/Saved/Config/LinuxServer/ MODIFY PalWorldSettings.ini
palworld/Pal/Saved/Config/LinuxServer/ MODIFY Engine.ini
palworld/Pal/Saved/Config/LinuxServer/ MODIFY Engine.ini
palworld/Pal/Saved/Config/LinuxServer/ MODIFY Game.ini
palworld/Pal/Saved/Config/LinuxServer/ MODIFY Game.ini
palworld/Pal/Saved/Config/LinuxServer/ MODIFY Input.ini
palworld/Pal/Saved/Config/LinuxServer/ MODIFY Input.ini
palworld/Pal/Saved/Config/LinuxServer/ MODIFY DeviceProfiles.ini
palworld/Pal/Saved/Config/LinuxServer/ MODIFY DeviceProfiles.ini
palworld/Pal/Saved/Config/LinuxServer/ MODIFY GameUserSettings.ini
palworld/Pal/Saved/Config/LinuxServer/ MODIFY GameUserSettings.ini
palworld/Pal/Saved/Config/LinuxServer/ MODIFY Scalability.ini
palworld/Pal/Saved/Config/LinuxServer/ MODIFY Scalability.ini
palworld/Pal/Saved/Config/LinuxServer/ MODIFY RuntimeOptions.ini
palworld/Pal/Saved/Config/LinuxServer/ MODIFY RuntimeOptions.ini
palworld/Pal/Saved/Config/LinuxServer/ MODIFY Hardware.ini
palworld/Pal/Saved/Config/LinuxServer/ MODIFY Hardware.ini
palworld/Pal/Saved/Config/LinuxServer/ MODIFY GameplayTags.ini
palworld/Pal/Saved/Config/LinuxServer/ MODIFY GameplayTags.ini
palworld/Pal/Saved/ImGui/ MODIFY Game.ini
palworld/Pal/Saved/ImGui/ MODIFY Game.ini
```

**Comment by Luatan** [comment]
In that case I'll wait until #356 is merged and then create a PR for it.

**Comment by Dashboy1998** [comment]
This can be closed as it was implemented in #412
